Make Introduction section and reflect IS-11 renaming (#60)
AMWA-TV/nmos-edid-connection-management: docs/EDID to Receiver Caps Mapping.md

{:toc}

## Introduction

The purpose of AMWA BCP-005-01 is to provide Best Current Practice guidelines for mapping EDID fields to Receiver Capabilities in the case Receiver is associated with an Output connected to a downstream counterpart which provides EDID.

The proposed mapping is affected when the Receiver is:

- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions)
- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)

> The Receiver SHALL include any types of conversion, omit any restrictions, or reflect the internal processing in own Receiver Capabilities.

Reviewer: Could you explain what this means.

Author: It is "SHALL" for circumstances described in the previous paragraph.
```
The proposed mapping is affected when the Receiver is:

- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions)
- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)
```

Maybe it makes sense to replace these two paragraphs with
```
The proposed mapping SHALL be affected when the Receiver is:

- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions)
- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)
```

Reviewer: Why not say it that way "The proposed mapping provides the mechanism for converting EDID information into Receiver Capabilities. The effective Receiver Capabilities MAY differ from those obtained from the proposed mapping when the Receiver is:
- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions) 
- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)"

Author: Thanks, that's great language.

The term 'Receiver' used in this document is defined in [IS-04][IS-04].

The terms 'Parameter Constraint' and 'Constraint Set' used in this document are defined in [BCP-004-01][BCP-004-01].

The term 'Output' used in this document is defined in [IS-11][IS-11].

BCP-005-01 is intended to be used in conjunction with an [IS-11][IS-11] and [BCP-004-01][BCP-004-01] deployment; however it has been written in such a way to provide useful functionality even in the absence of such a system.

This document is targeted against [E-EDID A2][E-EDID] which consist of EDID 1.4 (and covers EDID 1.3) and is referred to as _Base EDID_ and the [CTA-861-G][CTA-861] Extension Block imposed by HDMI. The information present in the EDID is subject to the requirements of the [Display Monitor Timing (DMT)][DMT] specification Version 1 Rev 12.

## Use of Normative Language

The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119][RFC-2119].

## Video Receivers

> If an [IS-04][IS-04] Video Receiver is associated with an Output which has EDID, its supported video formats MUST be mapped into Receiver's Capabilities according to the rules below.

Reviewer: If an [IS-04][IS-04] Video Receiver is associated with an Output which has an EDID, the optional mapping of the EDID supported video formats into Receiver's Capabilities SHALL be performed according to the rules below.

Author: ```suggestion
If an [IS-04][IS-04] Video Receiver is associated with an Output which has an EDID, the optional mapping of the EDID supported video formats into Receiver's Capabilities SHALL be performed according to the rules below.
```
